webbpsf/distortion.py

```diff
@@ -171,7 +179,12 @@ def distort_image(hdulist_or_filename, ext=0, to_frame='sci', fill_value=0,
         ynew += ynew_cen - np.median(ynew)
     
     # Convert requested coordinates to 'idl' coordinates
-    xnew_idl, ynew_idl = aper.convert(xnew, ynew, to_frame, 'idl')
+    if to_frame=='idl':
+        xnew_idl, ynew_idl = (xnew, ynew)
+    else:
+        # Choose conversion function
+        func_convert = getattr(aper, f'{to_frame}_to_idl')
+        xnew_idl, ynew_idl = func_convert(xnew, ynew)
```

Comment:
Same comment as above; this appears redundant with what `aper.convert` already does?

Reply:
Reverting to `aper.convert`.